packages/forklift-console-plugin/src/modules/Plans/planActions.tsx

```diff
@@ -292,11 +293,12 @@ DeleteModal.displayName = 'DeleteModal';
 const RestartModal = ({ plan, closeModal }: { plan: FlatPlan; closeModal: () => void }) => {
   const [isOpen, setIsOpen] = useState(true);
 
-  const exit = () => {
+  const exit = useCallback(() => {
     setIsOpen(false);
     closeModal();
-  };
-  const createMigrationMutation = useCreateMigrationMutation(plan.namespace);
+  }, [closeModal, setIsOpen]);
+
+  const createMigrationMutation = useCreateMigrationMutation(plan.namespace, exit);
```

Comment:
This is the actual fix right?

Reply:
yes. this is fix for the problem reported by @ahadas (added to the original issue)